src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,50 @@
+package duke.task;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * Represents a deadline type task.
+ */
+public class Deadline extends Task {
+
+    private final LocalDate time;
```

Comment:
```suggestion
    private final LocalDate date;
```
Perhaps it would be better to name this date, as it is a `LocalDate` object and not a `LocalTime` object? This could explain what the variable is more accurately.